Description: build flags support
 Don't force -O2 optimization. Make build logs verbose.
Author: Matthias Klose <doko@debian.org>
Forwarded: not-needed
Last-Update: 2013-10-30

--- a/gen-cc
+++ b/gen-cc
@@ -15,6 +15,11 @@
 	shift
 	shift
 	
+	echo $CC \$MODE -o \$BASE.o \$BASE.$EXT \\
+	        -MMD -MF \$DEPFILE -MP -MQ \$BASE.o \\
+	        $CPPFLAGS \\
+	        $CFLAGS \\
+	        "\$@"
 	$CC \$MODE -o \$BASE.o \$BASE.$EXT \\
 	        -MMD -MF \$DEPFILE -MP -MQ \$BASE.o \\
 	        $CPPFLAGS \\
--- a/wvrules-posix.mk
+++ b/wvrules-posix.mk
@@ -35,11 +35,6 @@
 # Default compiler we use for linking
 WVLINK_CC = $(CXX)
 
-ifneq ("$(enable_optimization)", "no")
-  CXXFLAGS+=-O2
-  CFLAGS+=-O2
-endif
-
 ifneq ("$(enable_warnings)", "no")
   CXXFLAGS+=-Wall -Woverloaded-virtual
   CFLAGS+=-Wall
